templates/web/zurich/admin/list_updates.html

```diff
@@ -7,10 +7,12 @@
 [%  internal_updates=[];
     sdm_notes=[];
     FOREACH update IN updates;
-        IF update.extra.is_internal_note;
-            internal_updates.unshift(update);
-        ELSE;
-            sdm_notes.unshift(update);
+        IF NOT update.extra.is_external_message;
```

Comment:
I think with this, having the update created at all isn't used for anything (beyond the fact it is some extra logging). Don't mind if it's left or not.